packages/core-data/src/selectors.ts

```diff
@@ -237,13 +250,28 @@ export function getEntityConfig(
  * @return Record.
  */
 export const getEntityRecord = createSelector(
-	(
+	function <
```

Comment:
did we intend to introduce `function` here? unless we have a strong reason to I would object to doing so because it changes the semantics of `this` inside the function. granted we don't use `this` here, but arrow function expressions hold an inherent safety that `function`s don't.